test/TestCases/FindingAndManipulatingElements.cpp

```diff
@@ -57,8 +61,12 @@ TEST_CASE ("find, findIf, findReverse, findIfReverse, contains", "[VctrBase][Fin
 
     REQUIRE (*c.findIf (beginsWith_t) == "two");
     REQUIRE (*d.findIf (beginsWith_t) == "two");
+    REQUIRE (c.indexIf (beginsWith_t).value() == 1);
+    REQUIRE (d.indexIf (beginsWith_t).value() == 1);
     REQUIRE (*c.findIfReverse (beginsWith_t) == "three");
     REQUIRE (*d.findIfReverse (beginsWith_t) == "three");
+    REQUIRE (c.indexIfReverse (beginsWith_t).value() == 2);
+    REQUIRE (d.indexIfReverse (beginsWith_t).value() == 2);
```

Comment:
Please add a test case to ensure correct behaviour if nothing is found.

Reply:
Good point, will add one!

Reply:
Added!